datacollection/openapi/X3/uam-data-collection-X3.yaml

```diff
@@ -464,7 +489,8 @@ components:
           sequence_id:
             description: >-
               	Integer indicating order of this waypoint in the list of waypoints.
-                must be sequential starting with 1.
+                must be sequential starting with 1.  If Operation is updated,
+                the sequence_id does not reset to 1.
```

Comment:
Points out a problem here. This is a model describing a single waypoint, so the only thing we can say here that is true in this domain is that sequence must be greater than or equal to 1, and that there are additional requirements imposed by the API endpoint.

Nick, Irene, how about this rewrite: "Integer indicating position of this waypoint in the ordered list of waypoints for the operation. See waypoint endpoint for more explanations".

We probably need to add a better description at the API endpoint, saying that the endpoint "Provides the sequence of waypoints for the operation. The sequence IDs for an operations waypoints must start at 1. Sequence IDs may not be repeated and there there may be no gaps in the sequence IDs for an operations waypoints. If an Operations waypoints are updated with subsequent submissions of waypoints to this endpoint, sequence IDs that already exist will be updated, and new sequence IDs will be added.

Reply:
good catch, pushing